library/summary.mli

```diff
@@ -63,7 +63,7 @@ end
     because its unfreeze may load ML code and hence add summary
     entries.  Thus is has to be recognizable, and handled properly.
    *)
-val declare_ml_modules_summary : (bool * string) list summary_declaration -> unit
+val declare_ml_modules_summary : (string option * string) list summary_declaration -> unit
```

Comment:
Can you describe the argments in the doc here?

Reply:
Done in #15991